openquake/hazardlib/source/non_parametric.py

```diff
@@ -125,8 +127,15 @@ def get_bounding_box(self, maxdist):
                 surfaces.extend(rup.surface.surfaces)
             else:
                 surfaces.append(rup.surface)
-        multi_surf = MultiSurface(surfaces)
-        west, east, north, south = multi_surf.get_bounding_box()
+        lons = []
+        lats = []
+        for surf in surfaces:
+            lo1, lo2, la1, la2 = surf.get_bounding_box()
+            lons.extend([lo1, lo2])
+            lats.extend([la1, la2])
+        # multi_surf = MultiSurface(surfaces)
+        # west, east, north, south = multi_surf.get_bounding_box()
+        west, east, north, south = get_spherical_bounding_box(lons, lats)
```

Comment:
Why commenting `multi_surf.get_bounding_box()` when it is doing exactly what you are doing?

Reply:
Because now you get this:
```
geo/surface/multi.py:141: in _set_tor
    raise ValueError(f"Surface {str(srfc)} not supported")
E   ValueError: Surface <openquake.hazardlib.geo.surface.gridded.GriddedSurface object at 0x12b0deca0> not supported
```
I will remove the commented part